vueui/src/components/view/cooking/microwave.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <template v-if="on">
+      <p>{{ cookingMessage }}</p>
+      <vui-progress :value="$root.$data.wtime" :min="start_time" :max="start_time + cook_time"/>
```

Comment:
progress component does not cap `value` value, so you need to add cap here or at progress component. Else it migth extend past it's limits. Like how vote timer counts a bit to negative numbers.